### PR TITLE
Add login page (that doesn't work yet)

### LINKED ISSUE #147: Create login page and component without API interaction
Page should look like https://petitions.moveon.org/login/index.html but doesn't need to submit yet.

### PR DESCRIPTION
**Pull request by jparkrr**
closes #147 

Next step would be #148, to make login work.

Again, there was a lot in common with https://github.com/MoveOnOrg/mop-frontend/blob/master/src/components/register-form.js

### CODE REVIEW

File: src/components/login-form.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.errors) {
+      this.setState({ presubmitErrors: null })
+    }
+    this.password.value = ''
+    this.passwordConfirm.value = ''
+  }
+
+  isValidEmail(email) {
```
Comment (sreynen) [inline src/components/login-form.js]: Let's move this to lib.js since we're using it in a few places now.

File: src/routes.js [hunk 2]
```diff
@@ -41,6 +42,7 @@ export const routes = (store) => {
         <Route path='/:organization/create_start.html' component={CreatePetitionPage} onEnter={orgLoader} />
         <Route path='/:organization/' component={Home} onEnter={orgLoader} />
         <Route path='/login/register.html' component={Register} />
+        <Route path='/login/index.html' component={Login} />
```
Comment (sreynen) [inline src/routes.js]: This should also respond to just `/login/`, as the current version does: https://petitions.moveon.org/login/ That might conflict with `/:organization/`. If so, we can actually remove `/:organization/`, as we recently decided it's no longer needed.

Comment (jparkrr) [reply src/routes.js]: Good point! I fixed the conflict by moving `/:organization/` after the new `/login/`.

**Comment by jparkrr** [comment]
Thanks for the review! Should be good now.

File: test/components/login-form.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+import LoginForm from '../../src/components/login-form'
+
+describe('<LoginForm />', () => {
+  it('displays fields', () => {
+    const login = shallow(<LoginForm />)
+    expect(login.find('input[name="email"]').length).to.equal(1)
+    expect(login.find('input[name="password"]').length).to.equal(1)
+  })
+
+  it.skip('does something different when logged in (TODO)', () => {})
```
Comment (sreynen) [inline test/components/login-form.js]: Does it actually do something different when logged in? https://petitions.moveon.org/login/ doesn't seem to.

**Pull request merged**